modules/apiserversource-kn.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="apiserversource-kn_context"]
+= Using the ApiServerSource with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete an ApiServerSource using `kn` commands.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have created the `default` broker in the same namespace that the ApiServerSource will be installed in.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. Create a service account, cluster role and cluster binding for the ApiServerSource.
++
+You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default <1>
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: event-watcher
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - get
+      - list
+      - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: k8s-ra-event-watcher
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: event-watcher
+subjects:
+  - kind: ServiceAccount
+    name: events-sa
+    namespace: default <1>
+----
++
+<1> Adjust this namespace to the namespace that you have selected for installing ApiServerSource.
++
+[NOTE]
+====
+If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
+====
++
+Create the service account, cluster role and cluster binding by entering the following command:
++
+----
+$ oc apply --filename authentication.yaml
+----
+
+. Create an ApiServerSource event source by entering the following command:
++
+----
+$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa
```

Comment:
It should add "--mode Resource" 

see  kn source apiserver create --help

      --mode string              The mode the receive adapter controller runs under:,
                                 "Ref" sends only the reference to the resource,
                                 "Resource" send the full resource. (default "Ref")

with the default "Ref" mode, the "The logs should contain lines similar to the following" would not show what it shows. It would just show plain Events without the involvedObject, which would not be useful in this case...